src/ImageSharp/Formats/WebP/LossyUtils.cs

```diff
@@ -0,0 +1,912 @@
+// Copyright (c) Six Labors and contributors.
+// Licensed under the Apache License, Version 2.0.
+
+using System;
+using System.Buffers.Binary;
+
+namespace SixLabors.ImageSharp.Formats.WebP
+{
+    internal static class LossyUtils
+    {
+        private static void Put16(int v, Span<byte> dst)
+        {
+            for (int j = 0; j < 16; ++j)
+            {
+                Memset(dst.Slice(j * WebPConstants.Bps), (byte)v, 0, 16);
+            }
+        }
```

Comment:
maybe faster:

```C#
for (int j = 0; j < 16 * WebPConstants.Bps; j+=WebPConstants.Bps) 
{
    Memset(dst.Slice(j), (byte)v, 0, 16);
}
```
this avoids the repeated multiplication.
(I assume WebPConstants.Bps is const here. If not, WebPConstants.Bps and 16*WebPConstants.Bps should be extracted to variables first).